python/dgl/distributed/kvstore.py

```diff
@@ -1394,6 +1394,12 @@ def count_nonzero(self, name):
             total += res.num_local_nonzero
         return total
 
+    @property
```

Comment:
how about using the same name in `KVServer` to be consistent?
https://github.com/dmlc/dgl/blob/72781efbcf2532254957090c889ec5b7b9563245/python/dgl/distributed/kvstore.py#L721-L724